go/api/pubsub_subscription_config.go

```diff
@@ -0,0 +1,202 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import (
+	"github.com/valkey-io/valkey-glide/go/api/errors"
+	"github.com/valkey-io/valkey-glide/go/protobuf"
+)
+
+// *** BaseSubscriptionConfig ***
+
+type ChannelMode interface {
```

Comment:
Why we need this?

Reply:
Do you miss a push?
I mean `Command() string` and `UnsubscribeCommand() string` are not used (and not needed)